docs/UserGuide.md

```diff
@@ -6,7 +6,7 @@ title: User Guide
 VolunSync is a **desktop app for keeping track of volunteers and volunteering events, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, VolunSync can get your event and personnel management tasks done faster than traditional GUI apps.
 
 * Table of Contents
-{:toc}
+  {:toc}
```

Comment:
should not indent `{:toc}` as jekyll will freak out and not display the entire table of contents.
Instead, this is what is displayed.
![image](https://github.com/user-attachments/assets/386f75aa-ef76-4081-a891-e76ac6fdd979)

Reply:
Fixed